Symfony/CS/Fixer/PSR2/VisibilityFixer.php

```diff
@@ -165,7 +165,7 @@ private function grabAttribsBeforePropertyToken(Tokens $tokens, $index)
      * @param array  $tokenAttribsMap token to attribute name map
      * @param array  $attribs         array of token attributes
      *
-     * @return array map of grabbed attributes, key is attribute name and value is array of index and clone of Token
```

Comment:
Actually, I'd prefer to see this comment moved to the "long description". Way more readable IMO.